README.md

```diff
@@ -78,6 +79,22 @@ A review object would look like this:
 - API response:
 A JSON-encoded **updated** review object with the new `likes_count`
 
+
+### `/newreview`
+
+- method: `POST`.
+- description: deletes selected review.
```

Comment:
Should say "add review", I think?

Reply:
No, `/newreview` is my endpoint for adding new review :)